src/utils/utils.js

```diff
@@ -32,6 +33,11 @@ export async function hash256 (hex: any) {
   return Promise.resolve(hash256Sync(hex))
 }
 
+export async function secp256k1Sign (message: Buffer, key: Buffer) {
+  const sigObj = await secp256k1.sign(message, key)
```

Comment:
Talk to Eliran about nonces. Doesn't this use https://tools.ietf.org/html/rfc6979 already, so we don't need to do anyting?

Reply:
Talked to Eliran about this, and it sounds like we are doing the nonces correctly.

Reply:
This is verified to be using rfc6979, so no need to change this.